### PR TITLE
[simd128.js]  1.Fix offset of SIMD.Float32x4.load by adding tarray.byteO...

### PR DESCRIPTION
**Pull request by chuan9**
...ffset.

              2.add test cases in the /test/mjsunit/simd/float32x4.js for 3155 bug

BUG=XWALK-3155

### CODE REVIEW

**Comment by huningxin** [comment]
@chuan9 please fix the coding style issue.

File: test/mjsunit/simd/float32x4.js [hunk 1]
```diff
@@ -942,3 +942,32 @@ function testArrayOfFloat32x4() {
...
+    assertEquals(i, v.x);
+    assertEquals(i+1, v.y);
+    assertEquals(i+2, v.z);
+    assertEquals(i+3, v.w);
+  }
+}
+
+testFloat32x4OveralignedLoad();
+
+function testFloat32x4ArrayIndex() {
```
Comment (huningxin) [inline test/mjsunit/simd/float32x4.js]: This test case is for another bug, correct? Please add it into the corresponding PR.

**Comment by huningxin** [comment]
@chuan9, please use one PR for one commit. It is easier to review. Thanks.

**Comment by chuan9** [comment]
git merge two PR automatically if the first PR is not accept

**Pull request closed without merging**